From de66d347f5126cf419dd91386d28d097841cc5d9 Mon Sep 17 00:00:00 2001
From: enkore <public@enkore.de>
Date: Sun, 4 Oct 2015 18:05:18 +0200
Subject: [PATCH] Fix abnomal formatting of authentication header lines

---
 src/polkitagent/polkitagenttextlistener.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/polkitagent/polkitagenttextlistener.c b/src/polkitagent/polkitagenttextlistener.c
index e63c285..d864dfb 100644
--- a/src/polkitagent/polkitagenttextlistener.c
+++ b/src/polkitagent/polkitagenttextlistener.c
@@ -206,9 +206,9 @@ on_completed (PolkitAgentSession *session,
 
   fprintf (listener->tty, "\x1B[1;31m");
   if (gained_authorization)
-    fprintf (listener->tty, "==== AUTHENTICATION COMPLETE ===\n");
+    fprintf (listener->tty, "==== AUTHENTICATION COMPLETE ====\n");
   else
-    fprintf (listener->tty, "==== AUTHENTICATION FAILED ===\n");
+    fprintf (listener->tty, "==== AUTHENTICATION FAILED ====\n");
   fprintf (listener->tty, "\x1B[0m");
   fflush (listener->tty);
 
@@ -473,7 +473,7 @@ polkit_agent_text_listener_initiate_authentication (PolkitAgentListener  *_liste
 
   fprintf (listener->tty, "\x1B[1;31m");
   fprintf (listener->tty,
-           "==== AUTHENTICATING FOR %s ===\n",
+           "==== AUTHENTICATING FOR %s ====\n",
            action_id);
   fprintf (listener->tty, "\x1B[0m");
   fprintf (listener->tty,
@@ -487,7 +487,7 @@ polkit_agent_text_listener_initiate_authentication (PolkitAgentListener  *_liste
       if (identity == NULL)
         {
           fprintf (listener->tty, "\x1B[1;31m");
-          fprintf (listener->tty, "==== AUTHENTICATION CANCELED ===\n");
+          fprintf (listener->tty, "==== AUTHENTICATION CANCELED ====\n");
           fprintf (listener->tty, "\x1B[0m");
           fflush (listener->tty);
           g_simple_async_result_set_error (simple,
-- 
1.9.5

