From 270da3e17c2aeca5d251a49b000b79f1a7b0a44f Mon Sep 17 00:00:00 2001
From: Jan Rybar <jrybar@redhat.com>
Date: Mon, 27 Jan 2020 18:08:37 +0000
Subject: [PATCH 5/5] Restoring terminal setting erases unread input

---
 src/programs/pkttyagent.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/programs/pkttyagent.c b/src/programs/pkttyagent.c
index 13879a2..8028f2d 100644
--- a/src/programs/pkttyagent.c
+++ b/src/programs/pkttyagent.c
@@ -56,7 +56,7 @@ static void tty_handler(int signal)
 
   if (tty_flags_saved)
   {
-    tcsetattr (fileno (tty), TCSAFLUSH, &ts);
+    tcsetattr (fileno (tty), TCSADRAIN, &ts);
   }
 
   kill(getpid(), signal);
-- 
2.25.0

