From 4ff1abe4a4c1f8c8378b9eaddb0346ac6448abd8 Mon Sep 17 00:00:00 2001
From: Jan Rybar <jrybar@redhat.com>
Date: Tue, 25 Jan 2022 15:38:53 +0000
Subject: [PATCH] SUID bit removed after chown

---
 meson_post_install.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson_post_install.py b/meson_post_install.py
index 0ab7469..e742310 100644
--- a/meson_post_install.py
+++ b/meson_post_install.py
@@ -30,8 +30,8 @@ except KeyError:
 dst = os.path.join(bindir, 'pkexec')
 
 if os.geteuid() == 0:
-    os.chmod(dst, 0o4755)
     os.chown(dst, 0, -1)
+    os.chmod(dst, 0o4755)
 else:
     print(
         'Owner and mode of {} need to be setuid root (04755) after '
@@ -63,8 +63,8 @@ for dst in dst_dirs:
 dst = os.path.join(pkglibdir, 'polkit-agent-helper-1')
 
 if os.geteuid() == 0:
-    os.chmod(dst, 0o4755)
     os.chown(dst, 0, -1)
+    os.chmod(dst, 0o4755)
 else:
     print(
         'Owner and mode of {} need to be setuid root (04755) after '
-- 
GitLab

