diff -up polkit-125/src/programs/pkexec.c.omv~ polkit-125/src/programs/pkexec.c
--- polkit-125/src/programs/pkexec.c.omv~	2025-01-13 00:20:02.534123533 +0100
+++ polkit-125/src/programs/pkexec.c	2025-01-13 00:20:44.726766846 +0100
@@ -207,7 +207,7 @@ open_session (const gchar *user_to_auth,
 	    {
 	      const char *eq = strchr (envitem, '=');
 	      g_assert (eq);
-	      if (!xdg_runtime_dir_is_owned_by (eq + 1, target_uid))
+	      if (target_uid != 0 && !xdg_runtime_dir_is_owned_by (eq + 1, target_uid))
 		continue;
 	    }
 
