diff -up polkit-0.117/src/polkitbackend/polkitbackendjsauthority.cpp.3~ polkit-0.117/src/polkitbackend/polkitbackendjsauthority.cpp
--- polkit-0.117/src/polkitbackend/polkitbackendjsauthority.cpp.3~	2020-08-28 03:08:08.549773068 +0200
+++ polkit-0.117/src/polkitbackend/polkitbackendjsauthority.cpp	2020-08-28 03:14:13.626262740 +0200
@@ -1198,7 +1198,7 @@ polkit_backend_js_authority_get_admin_au
   if (ret == NULL)
     ret = g_list_prepend (ret, polkit_unix_user_new (0));
 
-  JS_MaybeGC (authority->priv->cx);
+  JS_GC (authority->priv->cx);
 
   return ret;
 }
@@ -1295,7 +1295,7 @@ polkit_backend_js_authority_check_author
   if (!good)
     ret = POLKIT_IMPLICIT_AUTHORIZATION_NOT_AUTHORIZED;
 
-  JS_MaybeGC (authority->priv->cx);
+  JS_GC (authority->priv->cx);
 
   return ret;
 }
