diff -up ./src/polkitagent/polkitagentlistener.c.ori ./src/polkitagent/polkitagentlistener.c
--- ./src/polkitagent/polkitagentlistener.c.ori	2018-07-20 13:57:11.435302272 +0200
+++ ./src/polkitagent/polkitagentlistener.c	2018-07-20 13:57:30.427321200 +0200
@@ -439,6 +439,7 @@ polkit_agent_listener_register_with_opti
           server->thread_initialization_error = NULL;
           g_thread_join (server->thread);
           server_free (server);
+          server = NULL;
           goto out;
         }
     }
