diff -Naur polkit-0.112/src/polkitbackend/polkitbackendactionpool.c polkit-0.112.tpg/src/polkitbackend/polkitbackendactionpool.c
--- polkit-0.112/src/polkitbackend/polkitbackendactionpool.c	2013-04-29 17:28:57.000000000 +0000
+++ polkit-0.112.tpg/src/polkitbackend/polkitbackendactionpool.c	2015-03-15 10:26:48.701309305 +0000
@@ -1003,7 +1003,7 @@
         action->implicit_authorization_inactive = pd->implicit_authorization_inactive;
         action->implicit_authorization_active = pd->implicit_authorization_active;
 
-        g_hash_table_insert (priv->parsed_actions, action->action_id, action);
+        g_hash_table_replace (priv->parsed_actions, action->action_id, action);
 
         /* we steal these hash tables */
         pd->annotations = NULL;
